source/target/nxp/lpc4088/target_lpc4088dm.c

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file    target.c
+ * @brief   Target information for the lpc4088
+ *
+ * DAPLink Interface Firmware
+ * Copyright (c) 2009-2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "target_config.h"
+
+// The file flash_blob.c must only be included in target.c
+#include "flash_blob.c"
+
+// The LPC4088 Display Module has 512K internal flash and 16M external QSPI flash
+const target_cfg_t target_device = {
+	.sector_size    = KB(4),
+	.sector_cnt     = ((MB(16) + KB(512)) / KB(4)),
+	.flash_start    = 0,
+	.flash_end      = MB(16) + KB(512),
```

Comment:
I just noticed that the flash algo has support for this:
```
    } else if (adr >= 0x80000) {
        /* This happens when a combined binary is flashed. The combined binary starts
           with 512kB that goes into the internal flash. After the 512kB comes the
           data to be written at the start of the SPIFI. SPIFI is erased on a
           need-to basis and not here. */
        return saveInSpifi(adr - 0x80000, sz, buf);
    }
```

This should work, so ignore my previous comment.